radixdlt-core/radixdlt/src/main/java/com/radixdlt/network/messaging/MessageCentralImpl.java

```diff
@@ -172,6 +183,10 @@
 					return Optional.empty();
 				},
 				messageFromPeer -> {
+					final var messageProcessingTime = System.currentTimeMillis() - processingStart;
+					avgMessageProcessingTime.update(messageProcessingTime);
+					totalMessageProcessingTime = Math.max(totalMessageProcessingTime + messageProcessingTime, 0L);
+					updateCounters();
 					if (log.isTraceEnabled()) {
```

Comment:
I think that lambda is long enough to deserve a dedicated method.

Reply:
I think it's easier to read to have it in one place in this case. It would seem a bit strange to have a separate method like: `logAndUpdateCountersForProcessedMessage(MessageFromPeer msg, Stopwatch processingStopwatch)`

Reply:
something like `logReceivedMessage()` would work too

Reply:
done